src/cf-notification/src/cf-notification.less

```diff
@@ -0,0 +1,44 @@
+/* ==========================================================================
+   Capital Framework
+   Notifications Styling
+   ========================================================================== */
+
+//
+// Theme variables
+//
+
+// Color variables
+
+@notification-bg:             #d2d3d5;
```

Comment:
These are `@m-notification-…` in cfgov-refresh. Do we want the atomic prefix on variables?